api/proto/teleport/legacy/types/types.proto

```diff
@@ -5984,6 +5986,11 @@ message PluginEntraIDSyncSettings {
   repeated string default_owners = 1;
 }
 
+message PluginSCIMSettings {
+  option (gogoproto.equal) = true;
+  string sso_connector_id = 1;
```

Comment:
Nit: Is this the connector's "metadata.name" basically? Maybe call it `sso_connector_name` then?